### PR TITLE
Fixed user Singleton  

### PR DESCRIPTION
**Pull request by Bahaaib**
Track Singleton issue here: 
https://github.com/ammaratef45/Attendance/issues/32

&

Track the commented tests issue here:
https://github.com/ammaratef45/Attendance/issues/30

### CODE REVIEW

**Comment by 0crat** [comment]
@ammaratef45[/z](https://www.0crat.com/u/ammaratef45) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#42](https://github.com/ammaratef45/attendance/issues/42) is now in scope, role is `REV`

File: attendance/lib/BackEnd/user.dart [hunk 1]
```diff
@@ -89,13 +91,9 @@ class User {
...
-
-  }
+  void _persist() {}
 
-  void _markSaved() {
-
-  }
+  void _markSaved() {}
 
 // @todo #30 check Singleton potential leak
```
Comment (ammaratef45) [inline attendance/lib/BackEnd/user.dart]: @Bahaaib please delete the puzzle

Comment (Bahaaib) [reply attendance/lib/BackEnd/user.dart]: Oops. Sure!

**Comment by ammaratef45** [comment]
@rultor merge

**Comment by ammaratef45** [comment]
@Bahaaib Thanks

**Comment by ammaratef45** [comment]
@rultor merge

**Comment by ammaratef45** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ammaratef45[/z](https://www.0crat.com/u/ammaratef45) everybody who has role `REV` is banned at [#42](https://github.com/ammaratef45/attendance/issues/42); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0pdd** [comment]
@Bahaaib the puzzle [#43](https://github.com/ammaratef45/Attendance/issues/43) is still not solved.

**Comment by 0crat** [comment]
Job `gh:ammaratef45/attendance#42` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#42](https://github.com/ammaratef45/attendance/issues/42) is now out of scope

**Comment by 0pdd** [comment]
@Bahaaib the only puzzle [#43](https://github.com/ammaratef45/Attendance/issues/43) is solved here.

**Pull request merged**